### PR TITLE
`overlap()` return all `physicsId`s.

### PR DESCRIPTION
**Pull request by gonnavis**
Now `overlap()` only return the first encountered object/actor and immediately break the `for` loop.
Though fast but the returned one `physicsId` based on `actors`' order.
So use it to filter out object is not stable.
Such as in `marching-cube.scn` the `npcPlayer`'s id is `5` and the `terrain`'s id is `6`, thus will wrongly stopped.
image

So one solution is let `overlap()` run full `for` loop and return all `physicsId`s, what is this PR done.
Sibling PR: https://github.com/webaverse/app-wasm/pull/29

---
I also tried transfer filterId(s) to wasm, and keep the `for` loop's `break`, but don't know why both failed.
https://github.com/gonnavis/app/tree/overlapFilter
https://github.com/gonnavis/app-wasm/tree/overlapFilter
https://github.com/gonnavis/app/tree/overlapFilters
https://github.com/gonnavis/app-wasm/tree/overlapFilters
And feel it's hard to say which has better prformance, it's based on scene.
So just made this one "return all physicsId" PR.

### CODE REVIEW

**Review by avaer — APPROVED** [review]
Makes sense.

**Comment by avaer** [comment]
Some arguments refactoring recommended on the WebAssembly side: https://github.com/webaverse/app-wasm/pull/29#pullrequestreview-880844367

File: physx.js [hunk 1]
```diff
@@ -833,12 +832,20 @@ const physxWorker = (() => {
       meshPositionOffset,
       meshQuaternionOffset,
       hitOffset,
-      idOffset,
     );
 
-    return scratchStack.u32[14] ? {
-      objectId: scratchStack.u32[15],
-    } : null;
+    let head = outputBufferOffset / 4;
```
Comment (avaer) [inline physx.js:837]: Minor, but rather than hardcoded `4`, we can use `Float32Array.BYTES_PER_ELEMENT`.

Comment (gonnavis) [reply physx.js]: Changed.

**Review by avaer — APPROVED** [review]

**Pull request merged**